Dynatrace/dynatrace-operator
# Uses arch.Flavor when annotation is not set

# Description
Webhook no longer sets a default value for flavor, and if not set the standalone init uses the `arch.Flavor`

## How can this be tested?
Deploy anything, not set the flavor annotation, then on a amd node multidistro should be used, while in an arm node default


## Checklist
